packages/ffe-grid-react/src/Grid.tsx

```diff
@@ -1,25 +1,59 @@
 import React, { ElementType } from 'react';
 import classNames from 'classnames';
-import { ComponentWithoutRefAsPropParams } from './types';
+import { ComponentWithoutRefAsPropParams, Gap } from './types';
+
+type SizeModifier = GridGapSize | Gap;
+
+interface GridGapSize {
+    gap?: Gap;
+}
 
 export type GridProps<As extends ElementType = any> =
     ComponentWithoutRefAsPropParams<As> & {
         /** Specify the internal gutter of the grid */
-        gap?: 'none' | '2xs' | 'xs' | 'sm' | 'md' | 'lg';
+        gap?: Gap;
+        /** Size modifiers for small screen sizes */
+        sm?: SizeModifier;
+        /** Size modifiers for medium screen sizes */
+        md?: SizeModifier;
+        /** Size modifiers for large screen sizes */
+        lg?: SizeModifier;
     };
 
+const sizeClasses = (size: 'none' | 'sm' | 'md' | 'lg', def?: SizeModifier) => {
+    if (def === undefined) {
+        return null;
+    } else if (typeof def === 'number' || typeof def === 'string') {
+        return `ffe-grid--gap-${size}-${def}`;
+    }
+
+    return classNames({
+        [`ffe-grid--${size}-gap-${def.gap}`]: def.gap || def.gap === 0,
```

Comment:
```suggestion
        [`ffe-grid--${size}-gap-${def.gap}`]: def.gap,
```
def.gap kan ikke vare ett number.